.github/policies/eventResponder.yml

```diff
@@ -212,3 +212,23 @@ configuration:
                 > **To indicate this PR needs the core team''s attention, apply the "Needs: Core Team 🧞" label!**
                 >
                 > The core team will only review and approve PRs that have this label applied!
+
+      - description: "ITA27 - AVM Bicep GitHub Workflows For All Modules Will Be Triggered"
```

Comment:
I'm fine with this. Too bad it won't block the merge like a file comment does, but anyway way better than nothing.
@AlexanderSehr anything to add?